From alan@linux.intel.com  Wed Feb 17 15:00:23 2010
From: Alan Cox <alan@linux.intel.com>
Date: Mon, 08 Feb 2010 10:03:58 +0000
Subject: serial: timberdale: Remove dependancies
To: greg@kroah.com, linux-kernel@vger.kernel.org
Message-ID: <20100208100353.2633.42797.stgit@localhost.localdomain>


MFD_TIMBERDALE doesn't appear to be defined anywhere. However the code in
question can build happily without platform specifics so remove the check

Signed-off-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/serial/Kconfig |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/serial/Kconfig
+++ b/drivers/serial/Kconfig
@@ -1449,7 +1449,6 @@ config SERIAL_BFIN_SPORT3_UART
 
 config SERIAL_TIMBERDALE
 	tristate "Support for timberdale UART"
-	depends on MFD_TIMBERDALE
 	select SERIAL_CORE
 	---help---
 	Add support for UART controller on timberdale.
